### PR TITLE
Password Policy: Add validation labels to Update Password screen

### PR DESCRIPTION
**Pull request by linoman**
**What is this feature?**

This PR is needed to display validation labels when updating the password. These labels help guide the user to determine whether the new password complies with the Strong Password policy (when enabled).

**Why do we need this feature?**

This PR is part of the front-end implementation of the Strong Password Policy.

Password Policy checklist
https://github.com/grafana/identity-access-team/issues/562

**Who is this feature for?**

IAM.

**Which issue(s) does this PR fix?**:

Fixes https://github.com/grafana/identity-access-team/issues/590

**Special notes for your reviewer:**

- Pristine
image

- Dirty
image

- Validation upon submit
image

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: public/app/core/components/ForgottenPassword/ChangePassword.tsx [hunk 1]
```diff
@@ -19,6 +25,10 @@ interface PasswordDTO {
 
 export const ChangePassword = ({ onSubmit, onSkip, showDefaultPasswordWarning }: Props) => {
   const styles = useStyles2(getStyles);
+  const [displayValidationLabels, setDisplayValidationLabels] = useState(false);
+  const [pristine, setPristine] = useState(true);
+  const [newPassword, setNewPassword] = useState('');
```
Comment (Clarity-89) [inline public/app/core/components/ForgottenPassword/ChangePassword.tsx:30]: You don't need to do this, can get the up-to-date value from `watch` instead: 
```
const {handleSubmit, watch, ... } = useForm();
const newPassword = watch('newPassword');
```

**Review by vesnadean — APPROVED** [review]
Nice work!

File: public/app/core/components/ForgottenPassword/ChangePassword.tsx (hunk 1 above)
Comment (linoman) [reply public/app/core/components/ForgottenPassword/ChangePassword.tsx]: I have changed this as requested. I've also changed this for `public/app/features/profile/ChangePasswordForm.tsx`.

**Review by Clarity-89 — APPROVED** [review]

File: public/app/features/profile/ChangePasswordForm.tsx [hunk 2]
```diff
@@ -53,7 +52,8 @@ export const ChangePasswordForm = ({ user, onChangePassword, isSaving }: Props)
       `}
     >
       <Form onSubmit={onChangePassword}>
```
Comment (Clarity-89) [inline public/app/features/profile/ChangePasswordForm.tsx:54]:
```suggestion
      <Form onSubmit={onChangePassword} maxWidth={400}>
```
Not related to your PR, but could you remove the wrapper div and put the max width on the Form itself. Also could you update the Form import to be `import { Form } from 'app/core/components/Form/Form';` as importing it from grafana/ui is deprecated?

Comment (linoman) [reply public/app/features/profile/ChangePasswordForm.tsx]: Changes as requested

**Pull request merged**